### DISCUSSION TITLE
Which Method to Update Cloudflared

### DISCUSSION THREAD

**Discussion by gcortes** [tteck/Proxmox#2443] [Q&A] [answered]
I used your script to install cloudflared. It's working fine. I'm now  getting an update message on my cloudflare dashboard. I found these instructions to do the update.

`If installed via apt:

Update the cloudflared package:
sudo apt-get upgrade cloudflared

Restart the service:
sudo systemctl restart cloudflared.service

If installed manually via dpkg -i:

You can check if cloudflared was installed by a package manager by running ls -la /usr/local/etc/cloudflared/ and looking for .installedFromPackageManager in the output.

Update the cloudflared package:
curl -L --output cloudflared.deb https://github.com/cloudflare/cloudflared/releases/latest/download/cloudflared-linux-amd64.deb && sudo dpkg -i cloudflared.deb

Restart the service:
sudo systemctl restart cloudflared.service`

When I checked for .installedFromPackageManager, it was present. Did your script install using dpkg? I'm asking because I don't see any mention of it in the script. It may just be the case that I don't know what to look for.

**Comment by tteck** [comment, marked as answer]
Cloudflared is updated by a regular `apt update && apt upgrade` also the Proxmox VE LXC Updater can get it done 😉
